main.go

```diff
@@ -0,0 +1,457 @@
+/*
+Code is taken and modified from:
+github.com/influxdata/telegraf
+*/
+
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/influxdata/telegraf/config"
```

Comment:
why we need this?

Reply:
crypto-x509 is the x509 struct,
we use aws-lambda to provide context to the app,
telegraf config is used in the x509 struct.